build.py

```diff
@@ -48,9 +53,12 @@ def copy_artifacts():
         execute("cp ./_spidermonkey_install/lib/libmozjs* ./python/pythonmonkey/")
 
 def build():
+    execute("git submodule update --init --recursive")
     ensure_spidermonkey()
     run_cmake_build()
     copy_artifacts()
+    os.chdir(TOP_DIR + "/python/pminit")
+    execute(sys.executable + " ./post-install-hook.py")
```

Comment:
This will work for devs and as long as we don't remove the dependency or change the pminit dependency.

Reply:
Is this against the point that we want to `npm i` at installation time?